cmd/podman/images/scp.go

```diff
@@ -147,6 +147,16 @@ func scp(cmd *cobra.Command, args []string) (finalErr error) {
 		return err
 	}
 
+	allLocal := true // if we are all localhost, do not validate connections but if we are using one localhost and one non we need to use sshd
+	for _, val := range cliConnections {
+		if !strings.Contains(val, "@localhost::") {
+			allLocal = false
```

Comment:
Why do you need allLocal, just set the
```
if !strings.Contains(val, "@localhost::") {
	cliConnections = []string{}
        break
}
```

Reply:
well we only want to clear cliConnections if both of the sources are @ localhost:: because if one is localhost and one isn't we still need to validate the connections.

Reply:
ok,

Reply:
Well at least break as soon as you set it false.